<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>autoriz</class>
 <widget class="QWidget" name="autoriz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>autoriz</string>
  </property>
  <widget class="QPushButton" name="enter">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pass_le">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="login_le">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Pass</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
